src/types.ts

```diff
@@ -117,8 +117,8 @@ export interface Section {
 export interface Question {
   id: string;
   label?: string;
-  value?:string;
-  type: string;
+  value?: string;
```

Comment:
@Twiineenock I see `value` is of type `any` in form-engine and not `string`

Reply:
@NethmiRodrigo I had set the type to string because of this `eslint` error `Unexpected any. Specify a different type.eslint[@typescript-eslint/no-explicit-any](https://typescript-eslint.io/rules/no-explicit-any)`

Reply:
Should we go for `any` or `string`?

Reply:
No worries! Added `/ eslint-disable-next-line @typescript-eslint/no-explicit-any` to escape `eslint` check for `any` when validating question `value` attribute.